Closes #3

killumslow opened the pull request:
Basic layout to initially follow until further research goes into each topic and sub-topic. Will generate issues for each.

Commit:
initial mission, goals and vision statement

Commit:
mission and vision statement

Commit:
Updated based on pull request feedback

Commit:
initial basic framework

Commit:
update

bgordon134 commented on content/culture.md:
```diff
@@ -0,0 +1,12 @@
+# Culture
+
+## implementation methods that affect diversity and inclusion
```
Implementation

bgordon134 commented on content/culture.md:
```diff
@@ -0,0 +1,12 @@
+Methodologies
+
+
+## ways to establish and maintain organizational culture
```
Ways

bgordon134 commented on content/culture.md:
```diff
@@ -0,0 +1,12 @@
+
+## ways to establish and maintain organizational culture
+
+responsibilities of all employees
```
Responsibilities

bgordon134 commented on content/culture.md:
```diff
@@ -0,0 +1,12 @@
+
+responsibilities of all employees
+
+responsibilities of management
```
Responsibilities

bgordon134 commented on content/recruitment.md:
```diff
@@ -0,0 +1,13 @@
+# Recruitment
+
+
+## implementation methods that affect diversity and inclusion
```
Implementation

bgordon134 commented on content/recruitment.md:
```diff
@@ -0,0 +1,13 @@
+Methodologies
+
+
+## ways to establish and maintain employees
```
Ways

bgordon134 commented on content/recruitment.md:
```diff
@@ -0,0 +1,13 @@
+
+## ways to establish and maintain employees
+
+employee engagement
```
Employee

bgordon134 commented on content/recruitment.md:
```diff
@@ -0,0 +1,13 @@
+
+employee engagement
+
+responsibilities of management
```
Responsibilities

bgordon134 commented on content/retention.md:
```diff
@@ -0,0 +1,12 @@
+# Retention
+
+## implementation methods that affect diversity and inclusion
```
Implementation

bgordon134 commented on content/retention.md:
```diff
@@ -0,0 +1,12 @@
+Methodologies
+
+
+## ways to establish and maintain organizational culture
```
Ways

bgordon134 commented on content/retention.md:
```diff
@@ -0,0 +1,12 @@
+
+## ways to establish and maintain organizational culture
+
+responsibilities of all employees
```
Responsibilities

bgordon134 commented on content/retention.md:
```diff
@@ -0,0 +1,12 @@
+
+responsibilities of all employees
+
+responsibilities of management
```
Responsibilities

bgordon134 commented on content/diversity_and_inclusion.md:
```diff
@@ -0,0 +1,9 @@
+
+# Our framework
+
+## The four pillars
```
Is this section where we're going to describe what each of the pillars are with their goals, do you want that to be under each pillar respectively, or both?

killumslow replied:
I was thinking this section about how the four pillars flow, dependent on each other, from a high level. As we potentially add more pillars having a section that explains how they interact becomes more important, I think. Will add a sub-description explaining that if you agree. Otherwise we can remove and have them self contained.

bgordon134 replied:
Agree to the sub-description. Let me know if you need any help on drafting explanations.

bgordon134 reviewed (commented):
Biggest question is whether or not we want to have definitions for each pillar and where we want those to be housed.

killumslow committed:
Update diversity_and_inclusion.md

Added description to the four pillars section to provide better guidance on what is expected.

killumslow committed:
Update diversity_and_inclusion.md

added additional descriptions

bgordon134 commented on content/results.md:
```diff
@@ -0,0 +1,13 @@
+
+## ways to ensure true indicators are followed
+
+responsibilities of management
```
Responsibilities

bgordon134 commented on content/results.md:
```diff
@@ -0,0 +1,13 @@
+Methodologies
+
+
+## ways to ensure true indicators are followed
```
Ways

bgordon134 commented on content/diversity_and_inclusion.md:
```diff
@@ -0,0 +1,15 @@
+- Pillar dependencies
+    what must be in place for a successful implemenetation
+- How the pillars interact
+    Nature transitions between the pillars and how they should feed one another
```
Make "Nature of transitions" it's own sub-bullet.

bgordon134 commented on content/diversity_and_inclusion.md:
```diff
@@ -0,0 +1,15 @@
+- High level description of what the pillars are
+    taken from pillar documents
+- Pillar dependencies
+    what must be in place for a successful implemenetation
```
Pillar dependencies: what must be in place..
